### DISCUSSION TITLE
Permission issue with Termux & Python Script

### DISCUSSION THREAD

**Discussion by ElroyCrasto** [termux/termux-app#2514] [Q&A] [answered]
Installation using Curl Installer :

![SmartSelect_20220109-082633_Termux](https://user-images.githubusercontent.com/75974309/148667599-bde78976-ded9-4462-a155-d3cbca1b2967.jpg)

Running script According to guide :

![SmartSelect_20220109-082534_Termux](https://user-images.githubusercontent.com/75974309/148667605-58385724-a038-4ead-bfe1-f565e4ced481.jpg)

I've been stuck at this all night trying to figure out the problem.
At first I thought it was a permission issue so I changed the ownership as well as the access perms
But that didn't change the result.
Next After Looking around I tried to Reinstall Termux and Rechecked my Root still no progress. 

If anyone knows what i can do to fix this please let me know

I'm using a Galaxy Note 8 (Magisk-Rooted)
Heres Proof:   
![SmartSelect_20220109-084253](https://user-images.githubusercontent.com/75974309/148667817-33d306e4-d8a9-4dd2-aa51-004132fc541d.jpg)
![SmartSelect_20220109-084230_Root Checker Basic](https://user-images.githubusercontent.com/75974309/148667823-a276b4a0-4183-439a-b746-50776827dfb2.jpg)

**Comment by sk0kanik** [comment, marked as answer]
You are using third party scripts. OneShot is not provided by Termux. You should report it to the author of those scripts instead.

**Reply by ElroyCrasto** [reply]
According to my research and the github repo of Oneshot, which is now read only it should work on Termux.
